spec/index.bs

```diff
@@ -4402,13 +4438,18 @@ which may be one of the following:
 
 <script type=idl>
 dictionary GPUCommandEncoderDescriptor : GPUObjectDescriptorBase {
+    GPUQueueKind kind = "general";
```

Comment:
So we would have exactly one queue family of each type?

Reply:
Correct, yes. That matches D3D12, and I think it matches the Vulkan adapter geometry *in practice*, i.e. a recent AMD card has 1 general queue, one compute queue family with 2 queues, and 1 transfer queue family with 2 queues.